### PR TITLE
Tooltip Component

### PR DESCRIPTION
**Pull request by jmophoto**
This adds a tooltip component with the following props:

- `children` (node): Text or JSX to be displayed inside the tooltip
- `icon` (string): Icon to be displayed to indicate a tooltip. Defaults to `info`
- `iconSize` (number): Size of the icon to be displayed: Defaults to `20`
- `placement` (string): Used to quickly position tooltip. Must be `top`, `right`, `bottom` or `left`
- `style` (object): Used to override component styles. Can pass `fill` in to set the icon color
- `tooltipStyle` (object): Used to override the tooltip styling. Can be used for custom positioning.

Top:
screen shot 2016-06-02 at 5 12 10 pm

Right:
screen shot 2016-06-02 at 5 11 36 pm

Bottom:
screen shot 2016-06-02 at 5 13 08 pm

Left:
screen shot 2016-06-02 at 5 11 54 pm

@derek-boman

### CODE REVIEW

File: src/components/Tooltip.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      case 'bottom':
+        return {
+          top: offSet,
+          left: '50%',
+          marginLeft: -(this.props.iconSize / 2 + width / 2)
+        };
+      default:
+        return null;
+    }
+  },
```
Comment (iheartrachie) [inline src/components/Tooltip.js]: This makes me happy. 🍪

**Comment by iheartrachie** [comment]
I like this! Is there a way to make it work with text instead of just an icon?

**Comment by jmophoto** [comment]
I hadn't thought of that use-case, but it would be added later. For now, most of what we need is icon-based.

**Comment by iheartrachie** [comment]
:shipit:

**Pull request merged**